.github/workflows/diff_report.yml

```diff
@@ -35,7 +35,16 @@ jobs:
       branch: ${{ steps.branch.outputs.ref }}
       
     steps:
-    
+     - uses: khan/pull-request-comment-trigger@master
+       name: React with rocket on run
+       with:
+         trigger: 'GitHub, generate report'
+         reaction: rocket
+         prefix_only: true
+       env:
+         GITHUB_TOKEN: '${{ secrets.GITHUB_TOKEN }}'
+     - run: ' '
```

Comment:
execution of empty will be always a puzzle.
please make
`echo We print it here for this action to work`

https://github.com/Khan/pull-request-comment-trigger

Reply:
Solved